doc/api/http.md

```diff
@@ -1805,14 +1805,18 @@ changes:
 
 * `statusCode` {number}
 * `statusMessage` {string}
-* `headers` {Object}
+* `headers` {Object|Array}
```

Comment:
This also needs a `changes:` entry (sorry for being a bit repetitive here… :slightly_smiling_face:)

Reply:
Fixed, PTAL